src/alt/store/StoreMixin.js

```diff
@@ -46,7 +46,9 @@ const StoreMixin = {
         const value = spec.local && spec.local(state, ...args)
         const shouldFetch = spec.shouldFetch
           ? spec.shouldFetch(state, ...args)
+          /*eslint-disable*/
```

Comment:
yeah but it's complaining that I'm not using triple equals. I think it's a bug in eslint.